src/main/java/ecdar/controllers/QueryController.java

```diff
@@ -27,10 +30,26 @@ public class QueryController implements Initializable {
     private Query query;
     private final Map<QueryType, SimpleBooleanProperty> queryTypeListElementsSelectedState = new HashMap<>();
     private final Tooltip noQueryTypeSetTooltip = new Tooltip("Please select a query type beneath the status icon");
+    @FXML
+    private JFXTextField queryText;
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
         initializeActionButton();
+
+        queryText.textProperty().addListener(new ChangeListener<String>() {
+            @Override
+            public void changed(ObservableValue<? extends String> observable, String oldValue, String newValue) {
+                updateQueryTextFieldString(newValue);
+            }
+        });
+
+    }
+
+    //Method for replacing "<=" and ">=" with its unicode character
+    public void updateQueryTextFieldString(String newValue) {
```

Comment:
Maybe call the function something else like replaceOperators or something that makes the comment redundant.